crates/ibc-types-core-client/Cargo.toml

```diff
@@ -54,7 +54,7 @@ bytes = { version = "1.2.1", default-features = false }
 cfg-if = { version = "1.0.0", optional = true }
 derive_more = { version = "0.99.17", default-features = false, features = ["from", "into", "display"] }
 displaydoc = { version = "0.2", default-features = false }
-ibc-proto = { version = "0.37.0", default-features = false }
+ibc-proto = { version = "0.37.0", default-features = false, features = ["serde"] }
```

Comment:
Not a huge deal, but we gate `serde` through a feature. Fixed in #71

Reply:
Ah! Thanks for the fix 🙏